tests/test-grammar-integration.cpp

```diff
@@ -800,6 +948,165 @@ static void test_json_schema() {
         }
     );
 
+    test_schema(
+        "min -123 max 42",
+        // Schema
+        R"""({
+            "type": "integer",
+            "minimum": -123,
+            "maximum": 42
+        })""",
+        // Passing strings
+        {
+            "-123",
+            "-13",
+            "-12",
+            "-2",
+            "-1",
+            "0",
+            "1",
+            "5",
+            "40",
+            "42",
+        },
+        // Failing strings
+        {
+            "-0123",
+            "-124",
+            "-1123",
+            "-200",
+            "43",
+            "123",
+            "0123",
+        }
+    );
+
+    test_schema(
+        "min 5 max 30",
+        // Schema
+        R"""({
+            "type": "integer",
+            "minimum": 5,
+            "maximum": 30
+        })""",
+        // Passing strings
+        {
+            "5",
+            "10",
+            "30",
+        },
+        // Failing strings
+        {
+            "05",
+            "4",
+            "-1",
+            "31",
+            "123",
+            "0123",
+        }
+    );
+
+    test_schema(
+        "min 2",
+        // Schema
+        R"""({
+            "type": "integer",
+            "minimum": 2
+        })""",
+        // Passing strings
+        {
+            "2",
+            "1234567890000000",
+        },
+        // Failing strings
+        {
+            "0",
+            "1",
+            "12345678900000000",
+        }
+    );
+
+    test_schema(
+        "min 0",
```

Comment:
This schema looks identical with the "simple min 0" test from line 154. Can they be merged together to shorten this file?

Reply:
👌